pos_receipt_replace_user_by_trigram/models/res_users.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2024 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class ResUsers(models.Model):
+    _inherit = "res.users"
+
+    trigram = fields.Char(compute="_compute_trigram")
```

Comment:
```suggestion
    pos_trigram = fields.Char(compute="_compute_pos_trigram")
```
.It is used only in Point of Sale, to avoid possible collision

Reply:
Alternatively, extract the trigram field and it's generation algorithm to a generic `base_user_trigram` and contribute to https://github.com/OCA/server-backend

Reply:
for now I would simply move the computation to a util func that could be moved later out of the module w/ less impact